src/providers/consensus/client.py

```diff
@@ -35,6 +36,16 @@ class ConsensusClient(HTTPProvider):
     API_GET_BLOCK_HEADER = '/eth/v1/beacon/headers/{}'
```

Comment:
I would remove `/` at the start to be the same as the rest of endpoints

Reply:
nice catch!